packages/headless/src/features/search-and-folding/search-and-folding-request.ts

```diff
@@ -23,11 +23,10 @@ export const buildSearchAndFoldingLoadCollectionRequest = async (
     locale: state.configuration.search.locale,
     debug: state.debug,
     tab: state.configuration.analytics.originLevel2,
-    referrer: state.configuration.analytics.originLevel3,
+    referrer: navigatorContext.referrer,
```

Comment:
Am I correct to assume that the originLevel3 property in the engine's analytics configuration will become completely unused?

Reply:
Yes, with the next analytics